src/include/daos_fs_sys.h

```diff
@@ -180,15 +180,17 @@ dfs_sys_access(dfs_sys_t *dfs_sys, const char *path, int mask, int flags);
 /**
  * Change permission access bits. Symlinks are dereferenced.
  *
- * \param[in]	dfs_sys Pointer to the mounted file system.
- * \param[in]	path	Link path of object.
- * \param[in]	mode	New permission access modes. For now, we don't support
- *			the sticky bit, setuid, and setgid.
+ * \param[in]	dfs_sys 	Pointer to the mounted file system.
+ * \param[in]	path		Link path of object.
+ * \param[in]	mode		New permission access modes. For now,
+ * 				we don't support the sticky bit, setuid,
+ * 				and setgid.
+ * \param[in]	ignore_unsup	Ignore unsupported filesystem features
  *
- * \return		0 on success, errno code on failure.
+ * \return				0 on success, errno code on failure.
  */
 int
-dfs_sys_chmod(dfs_sys_t *dfs_sys, const char *path, mode_t mode);
+dfs_sys_chmod(dfs_sys_t *dfs_sys, const char *path, mode_t mode, bool ignore_unsup);
```

Comment:
this is a major version API change (it breaks the existing API); so not acceptable as is.

Reply:
Right. For example, third-party mpifileutils uses this function. Which would require a recompile. And, unlikely but possible, some unknown user application.